go/kbfs/libkbfs/kbfs_ops_test.go

```diff
@@ -4283,6 +4283,21 @@ func TestKBFSOpsReset(t *testing.T) {
 	children, err = kbfsOps.GetDirChildren(ctx, rootNode)
 	require.NoError(t, err)
 	require.Len(t, children, 1)
+
+	t.Logf("Reset it back")
+	md.override = true
+	err = kbfsOps.Reset(ctx, h, &oldID)
+	require.NoError(t, err)
+	require.Equal(t, oldID, oldID)
```

Comment:
you're requiring that two of the same thing are equal. I think you meant to check that `Equal(t, oldID, h.TlfID())`?

Reply:
Great catch, thanks!